_layouts/default.html

```diff
@@ -8,7 +8,9 @@
 <body id="{{ page.sectionid }}" class="{{ page.body_class }}">
 <div class="header">
     <div class="width-limiter">
-        <a class="logo" href="/">Valkey</a>
+        <a href="/" class="logo">
+            <img src="/assets/img/Valkey-wordmark.svg" alt="Valkey Logo" width="180px" />
```

Comment:
For now this should probably just have the logo + the opensans "Valkey"

The wordmark was created before a bunch of stuff was settled: it's Valkey not ValKey and the font's hasn't been officially picked yet. 

I'd actually be okay if removed the wordmark file was removed for now.